projectJava/task2_calculator/src/CalculatorDL.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Scanner;
+
+public class CalculatorDL {
+    static void executeCalculation(double num1, double num2, char operator) {
+        double result;
+        switch (operator) {
+            case '+':
+                result = num1 + num2;
+                System.out.println(num1 + " + " + num2 + " = " + result);
```

Comment:
Please try to use printf for such cases: System.out.printf(%f + %f + %f, num1, num2, result) (here and below)